builder/qemu/config.go

```diff
@@ -235,9 +235,13 @@ type Config struct {
 	// add [ "-global", "virtio-pci.disable-modern=on" ] to `qemuargs` depending on the
 	// guest operating system.
 	//
-	// ~> For `whpx`, note that [Stefan Weil's QEMU for Windows distribution](https://qemu.weilnetz.de/w64/)
```

Comment:
Why is this removed?

Reply:
According to the commit that changes this, WHPX is supported by Stefan Weil's Windows qemu distribution, so in that sense I tend to agree that this can be removed from our docs.
I cannot test this however since I don't have a Windows machine on hand